state/cassandra/cassandra.go

```diff
@@ -286,7 +287,12 @@ func (c *Cassandra) Set(req *state.SetRequest) error {
 		session = sess
 	}
 
-	return session.Query("INSERT INTO ? (key, value) VALUES (?, ?)", c.table, req.Key, bt).Exec()
+	ttl, err := parseTTL(req.Metadata)
+	if err != nil {
+		return fmt.Errorf("error parsing TTL from Metadata: %s", err)
+	}
+
+	return session.Query("INSERT INTO ? (key, value) VALUES (?, ?) USING TTL ?", c.table, req.Key, bt, ttl).Exec()
```

Comment:
`0` is the value parsed if ttl is not present - default behavior. So, what is the behavior for `0` TTL in cassandra? Also, is it also in seconds?

Reply:
Yes it is seconds. And Cassandra says that 0 or any negative number means to persist the record. https://docs.datastax.com/en/cql-oss/3.3/cql/cql_using/useExpireExample.html

Reply:
Thanks for catching this. This was only correct when a TTL could not be set at the table level. I updated it now. Someone wishing to explicitly to persist an item needs to specify 0 or any negative number. It's completely passthrough now.